README.markdown

```diff
@@ -34,11 +34,12 @@ Writing Objective-C? Check out our [Objective-C Style Guide](https://github.com/
 
 ## Naming
 
-Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter.
+Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter. Global constants, however, should have a project prefix.
 
 **Preferred:**
 
 ```swift
+let XYZSomeGlobalConstant = 123.456
```

Comment:
I was using global constant because Swift doesn't yet support class variable. 

But it seems we can either use 
- computed variable
- static let ...

in the class.

```
class A {
    var constantPI = {
        return 3.141592
    }

    static let constantMile = 1.60934
}
```

and we can access `ClassA.constantPI` or `ClassA.constantMile`.

As far as I understand, Swift's module is not as accessible thing as in other languages. When you create a project, it will automatically package all the classes in the project modules so that naming convention in other projects will not conflict with your local convention. (i.e., you still need to deal with conflicts within your own project) I would be happy to be wrong :) This is one of the shortcomings in Swift IMO, if I understand situation correctly.